pkg/cmd/service/service_test.go

```diff
@@ -48,7 +48,7 @@ const (
 	openFGAGRPCAddr   = "localhost:8081"
 )
 
-func ensureServiceUp(t *testing.T, transportCredentials credentials.TransportCredentials) {
+func ensureServiceUp(t *testing.T, transportCredentials credentials.TransportCredentials, useHTTPToo bool) {
```

Comment:
I'd always spin it up so we don't need to have the context. With the introduction of the healthcheck we ensure we don't run into a race condition. Plus the upcoming work for having random ports assigned should help too.